src/main/java/io/confluent/connect/jdbc/source/TableQuerier.java

```diff
@@ -107,12 +109,24 @@ public boolean next() throws SQLException {
   public void reset(long now) {
     closeResultSetQuietly();
     closeStatementQuietly();
+    releaseLocksQuietly();
     // TODO: Can we cache this and quickly check that it's identical for the next query
     // instead of constructing from scratch since it's almost always the same
     schemaMapping = null;
     lastUpdate = now;
   }
 
+  private void releaseLocksQuietly() {
+    if (db != null) {
+      try {
+        db.commit();
+      } catch (SQLException e) {
+        // intentionally ignored
```

Comment:
maybe `log.warn()`?

Reply:
`// intentionally ignored` was the style at the time